docs/development-and-testing.md

```diff
@@ -1,34 +1,56 @@
 ## Development and Testing
-0. You would need to have `quay.io` account, running docker and kubernetes; and make sure `quay.io/<QUAY_USERNAME>/direct-csi` repository is `public`.
-1. Go to project root
+1. You would need to have `quay.io` account, running docker and kubernetes.
+2. Docker login to your `quay.io` account.
```

Comment:
Should instead be

"Login to your registry account using the following command. For instance, if your registry is quay.io..."

Reply:
Fixed in different way.